cluster-state-service/cmd/root.go

```diff
@@ -46,6 +47,7 @@ a localized data store, which provides you a near-real-time view of your cluster
 	}
 	// TODO: Fix the description
 	rootCmd.PersistentFlags().StringVar(&config.SQSQueueName, sqsQueueNameFlag, "", "SQS queue name")
+	rootCmd.PersistentFlags().StringVar(&config.KinesisStreamName, kinesisStreamNameFlag, "", "Kinesis stream name")
```

Comment:
Doesn't matter how you treat your commits before merging but we usually squash into one commit (as long as everything belongs together logically) and rebase when merging.